tests/setup/helpers.sh

```diff
@@ -27,7 +27,9 @@ export PASS_INDEX_SILENT
 
 # helpers
 LOG_FILE=$PASSWORD_STORE_DIR/tests.log
-export LOG_FILE
+CLIP_FILE=$PASSWORD_STORE_DIR/clipboard.log
+PATH="./tests/fake:$PATH"
+export LOG_FILE CLIP_FILE PATH DISPLAY
```

Comment:
`DISPLAY` is not set here

```suggestion
export LOG_FILE CLIP_FILE PATH
```